README.md

```diff
@@ -1,59 +1,307 @@
 
-The application supports:
-  - RSA with key up to 4096 bits 
-  - ECDSA with secp256k1
-  - EDDSA with Ed25519 curve 
-  - ECDH with secp256k1 and curve25519 curves
+GnuPG application for Ledger devices
 
+This application implements "The OpenPGP card" specification revision 3.3.
+This specification is available in *doc* directory and at <https://g10code.com/p-card.html>.
 
-This release has known missing parts (see also Add-on) :
+The application supports:
 
-   * Ledger Blue support
-   * Seed mode ON/OFF via apdu
+- RSA with key up to 4096 bits
+- ECDSA with secp256k1
+- EDDSA with Ed25519 curve
+- ECDH with secp256k1 and curve25519 curves
 
+This release has known missing parts (see also Add-on):
 
-## Installation and Usage
+- Seed mode ON/OFF via apdu
 
-See the full doc at https://github.com/LedgerHQ/blue-app-openpgp-card/blob/master/doc/user/blue-app-openpgp-card.pdf
+## Installation and Usage
 
+See the full doc at <https://github.com/LedgerHQ/openpgp-card-app/blob/master/doc/user/app-openpgp-card.pdf>
 
 ## Add-on
 
 The GnuPG application implements the following addon:
-  - serial modification
-  - on screen reset
-  - 3 independent key slots
-  - seeded key generation
 
-Technical specification is available at https://github.com/LedgerHQ/blue-app-openpgp-card/blob/master/doc/developper/gpgcard3.0-addon.rst
+- Serial modification
+- On screen reset
+- 3 independent key slots
+- Seeded key generation
+
+Technical specification is available at <https://github.com/LedgerHQ/openpgp-card-app/blob/master/doc/developer/gpgcard-addon.pdf>
 
-   
 ### Key slot
 
-"The OpenPGP card" specification specifies:
-  - 3 asymmetric keys : Signature, Decryption, Authentication
-  - 1 symmetric key
+The OpenPGP card specification indicates:
+
+- 3 asymmetric keys:
+  - Signature,
+  - Decryption
+  - Authentication
+- 1 symmetric key
 
-The blue application allow you to store 3 different key sets, named slot. Each slot contains the above 4 keys.
+The application allows you to store 3 different key sets, named slot. Each slot contains the above 4 keys.
 You can choose the active slot on the main screen.
 When installed the default slot is "1". You can change it in settings.
 
-   
-### seeded key generation
+### Seeded key generation
 
 A seeded mode is implemented in order to restore private keys on a new token.
-In this mode key material is generated from the global token seeded.
+In this mode, key material is generated from the global token seed.
 
-Please consider SEED mode as experimental.
+Also, a backup/restore mechanism is provided.
 
-More details to come... 
+Please consider SEED mode as experimental.
 
 ### On screen reset
 
 The application can be reset as if it was fresh installed. In settings, choose reset and confirm.
 
+## Quick start guide
```

Comment:
could be interesting to have a short user guide to use the app

Reply:
Everything is described in the doc/* files, and a ref to the documentation is given at the end of the README.
Not sure what to write more here...

Reply:
ok 👍